client/shaders/3d_secondstage/opengl_fragment.glsl

```diff
@@ -0,0 +1,34 @@
+uniform sampler2D baseTexture;
+uniform sampler2D normalTexture;
+uniform sampler2D ShadowMapSampler;
+
+#define rendered baseTexture
+#define normalmap normalTexture
+#define depthmap ShadowMapSampler
```

Comment:
Why `PascalCase`?

Reply:
Can you elaborate the concern?

Reply:
consistency, I'd assume

Reply:
Both of the inconsistent names are coming from existing code in Minetest, and fixing these names is out of scope for this specific PR. What I can do is introduce new uniforms for post-processing texture names.

Is there a reason why GLSL uses camel case for variables while C++ uses snake case?

Reply:
@kilbith There's no arguing here, just a question.
Am I able to? Sure, I am. Not sure I'd want to after such an attitude, though.
Besides, _everything_ is relevant to the code, if it's written in the code itself.